python/taichi/core/util.py

```diff
@@ -113,13 +113,39 @@ def format_plain_text(fn):
         f.write(formatted)
 
 
+def _find_clang_format_bin():
+    OKGREEN = '\033[92m'
```

Comment:
Not sure if this works on Windows. Maybe we can make use of [`colorama`](https://pypi.org/project/colorama/), which is already a Taichi dependency?

Reply:
I found a few posts recommending the cross-platform https://pypi.org/project/colorama/, not sure if we should add this DEP just for this usage, though.

Reply:
i decided not to go fancy and just removed the colors..

Reply:
Done

Reply:
LOL, cool!